code/modules/admin/debug_verbs.dm

```diff
@@ -542,3 +496,55 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 
 	log_admin("[key_name(usr)] updated the mob sprite of [key_name(H)].")
 	message_admins("[ADMIN_TPMONTY(usr)] updated the mob sprite of [ADMIN_TPMONTY(H)].")
+
+
+/datum/admins/proc/transfer_job_bans()
```

Comment:
Make this a /proc/ in ban_panel.dm. There is no reason this should be a regular button as it will at most be used once.

Reply:
Scratch that, this PR is on hold until my DB stuff gets merged, then it will get added into the legacy ones.